spec/factories/orchestration_stack.rb

```diff
@@ -25,4 +27,18 @@
       x.value = "1"
     end
   end
+
+  factory :orchestration_stack_parameter_openstack_infra_swift_replicas, :parent => :orchestration_stack_parameter_openstack_infra do
+    after :create do |x|
+      x.name = "SwiftReplicas"
+      x.value = "3"
+    end
```

Comment:
Don't use `after :create` here...just use normal values

```ruby
factory :orchestration_stack_parameter_openstack_infra_swift_replicas, :parent => :orchestration_stack_parameter_openstack_infra do
  name "SwiftReplicas"
  value "3"
end
```

Also, you may want to consider *not* putting the value here and putting it explicitly in the spec.